packages/cli/package.json

```diff
@@ -1,22 +1,17 @@
 {
   "name": "@appland/appmap",
-  "version": "3.1.0",
+  "version": "3.0.0",
```

Comment:
This is going down a minor version, is that expected?

Reply:
Version is managed by semantic-release, so I'm expecting this to get updated after it's merged.

Reply:
The latest live release is `3.1.0`, so this change should not be made.